### PR TITLE
[FIX] Add CE agent to output figure filename templates

### PR DESCRIPTION
**Pull request by jdkent**
supercedes #479 
addresses poldracklab/fmriprep#2033

### CODE REVIEW

File: niworkflows/reports/figures.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
             "pattern": "[_/\\\\]acq-([a-zA-Z0-9]+)"
         },
         {
-            "name": "ce",
+            "name": "ceagent",
```
Comment (effigies) [inline niworkflows/reports/figures.json]: Does this have any API effect?

Comment (jdkent) [reply niworkflows/reports/figures.json]: No, I do not believe so. This does not impact the recently added --bids-filter, and makes `figures.json` naming consistent with [`fmriprep.yml`](https://github.com/nipreps/niworkflows/blob/bfffaf8ea8e6fe1615be4a24089a5d750bd6e024/niworkflows/reports/fmriprep.yml#L36).

Comment (effigies) [reply niworkflows/reports/figures.json]: Okay. @oesteban Is this consistent with your understanding as well?

**Comment by effigies** [comment]
Actually, since #479 isn't merged, let's just make this the main PR, and we'll merge into `master` after the next release.

File: niworkflows/reports/figures.json (hunk 1 above)
Comment (oesteban) [reply niworkflows/reports/figures.json]: Yup, I also think this should not affect the API

**Comment by effigies** [comment]
I've tested with a synthetic dataset where I switched from `run-*` to `ce-*`, and fmriprep 20.0.4 + this PR works as expected.

Given @bbfrederick's comments, I'm going to go ahead and merge.

Thanks!

**Comment by bbfrederick** [comment]
Works for me.  I'll wait until the dust settles, sync up with the patched release version of niworkflows, and then put in a PR for the BIDS_NAME thing.  AFAIK it doesn't actually cause any problems at the moment, but it is wrong and it might bite somebody later if they decide to use it, so it's not currently urgent.

**Pull request merged**